### ISSUE TITLE
Document update suggestion about the range of dynamic code block 

### ISSUE THREAD

**Issue by Ehwaz** [playframework/twirl#95]
Hi, I'm noob in using twirl and now learning it from the document.

I found that whitespace concludes the range of dynamic code block specified by '@'.
For example,
```
@main (menuList)("Liked Posts")    // Incorrect result: It prints 'main' template function as a value
@for (menu <- menuList) { ... }    // Compilation error: It says "'(' expected but ')' found."
```
above codes don't work and I had to remove whitespaces after '@' like below.
```
@main(menuList)("Liked Posts")
@for(menu <- menuList) { ... }
```

How about mentioning the effect of whitespace in Twirl document?
I spent hours to figure out what's wrong and I don't want others to suffer from it.

Update:
I found a similar issue was suggested to play google group 3 years ago.
https://groups.google.com/forum/#!topic/play-framework/8Z_KctGKWm4
if modifying template parser is tough one, I think letting users to know about the restriction is a good alternative.

**Comment by gmethvin** [comment]
Yes, I think clarification would be useful there. PRs welcome.

**Comment by Ehwaz** [comment]
Okay then I'll add it to doc and make a PR for it.
Thanks for the quick reply!

**Comment by Ehwaz** [comment]
Fixed by #96.
